emails/grant-notice

```diff
@@ -1,4 +1,4 @@
-Hello, {{user.username}}. {{mod.user.username}} has asked you to co-author {{mod.name}} on Kerbal Stuff.
+Hello, {{user.username}}. {{mod.user.username}} has asked you to co-author {{mod.name}} on {{ site_name }}.
```

Comment:
No, because 'site-name' is the value in the config, we reassign them to 'site_name' internally when we pass the vars to flask